sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -65,56 +68,54 @@ or [Azure CLI][azure_cli_endpoint_lookup]:
 az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "endpoint"
 ```
 
-#### Types of credentials
-The `credential` parameter may be provided as a [AzureKeyCredential][azure-key-credential] from [azure-core][azure_core],
-or as a credential type from Azure Active Directory.
-See the full details regarding [authentication][cognitive_authentication] of cognitive services.
-
-1.  To use an [API key][cognitive_authentication_api_key],
-    pass the key as a string into an instance of `AzureKeyCredential("<api_key>")`.
-    The API key can be found in the Azure Portal or by running the following Azure CLI command:
-
-    ```az cognitiveservices account keys list --name "resource-name" --resource-group "resource-group-name"```
-
-    Use the key as the credential parameter to authenticate the client:
-    ```python
-    from azure.ai.formrecognizer import FormRecognizerClient
-    from azure.core.credentials import AzureKeyCredential
-
-    endpoint = "https://<region>.api.cognitive.microsoft.com/"
-    credential = AzureKeyCredential("<api_key>")
-    form_recognizer_client = FormRecognizerClient(endpoint, credential)
-    ```
-
-2. To use an [Azure Active Directory (AAD) token credential][cognitive_authentication_aad],
-   provide an instance of the desired credential type obtained from the
-   [azure-identity][azure_identity_credentials] library.
-   Note that regional endpoints do not support AAD authentication. Create a [custom subdomain][custom_subdomain]
-   name for your resource in order to use this type of authentication.
-
-   Authentication with AAD requires some initial setup:
-   * [Install azure-identity][install_azure_identity]
-   * [Register a new AAD application][register_aad_app]
-   * [Grant access][grant_role_access] to Form Recognizer by assigning the `"Cognitive Services User"` role to your service principal.
-
-   After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
-   As an example, [DefaultAzureCredential][default_azure_credential]
-   can be used to authenticate the client:
-
-   Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
-   `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
-
-   Use the returned token credential to authenticate the client:
-    ```python
-    from azure.identity import DefaultAzureCredential
-    from azure.ai.formrecognizer import FormRecognizerClient
-    token_credential = DefaultAzureCredential()
-
-    form_recognizer_client = FormRecognizerClient(
-        endpoint="https://<my-custom-subdomain>.cognitiveservices.azure.com/",
-        credential=token_credential
-    )
-    ```
+#### Get the API key
+
+The API key can be found in the Azure Portal or by running the following Azure CLI command:
```

Comment:
nit : in the Azure Portal under the "Keys and Endpoint"

Reply:
I've actually been advised against specifying exactly where to find information in the Portal since apparently menu options change / get renamed frequently enough to cause problems.

Reply:
Oh good to know, I will then change it Java!